gateway/conf/samples/root.yml

```diff
@@ -1,35 +1,35 @@
 $namespace avalanche:
   configurationPath: avalanche.yml
-  schemaPath: ethereum-schema.json
+  schemaFileName: ethereum-schema.json
```

Comment:
Keep the "schemaPath" name. By renaming it to "xxxFileName" you're giving an illusion that this cannot be a path.